release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -145,6 +145,13 @@ With this release, the resource limits for the descheduler operand are removed.
 [id="ocp-4-14-scalability-and-performance"]
 === Scalability and performance
 
+[id="ocp-4-14-PAO-image-must-gather"]
+==== PAO must-gather image added to default must-gather image
+
+With this release, the Performance Addon Operator (PAO) must-gather image is no longer required as an argument for the `must-gather` command to capture debugging data related to low-latency tuning. The functions of the PAO must-gather image are now under the default plugin image used by the `must-gather` command without any image arguments.
+
+For further information about gathering debugging information relating to low-latency tuning, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support_cnf-master[Collecting low latency tuning debugging data for Red Hat Support]
```

Comment:
Add a period after the xref.

Reply:
You'll be sorry you asked!!! 
In our Telco team we have a slightly different RN workflow at the moment. Strictly speaking we add the RN text to the story, get approvals, then a single person from the team creates a PR with all the RNs for a release and creates a PR. 
However, the workflow is a bit up in the air because there's some debate if it's a good workflow. For now, we link all our RN PRs to a single tracking task in JIRA ([example](https://issues.redhat.com/browse/TELCODOCS-1155)) and the RN person just checks nothing has been missed. It's just easier to scan RN only PRs I guess.

Also just wondering... my release note is only for the upcoming release branch but the procedure is going on main too. Wouldn't that cause a problem?

Reply:
@rohennes, yeah, you'd need a separate PR for just the 4.14 RN, as done here.